### ISSUE TITLE
The multiple service nodes in Session's decentralized network are not changing.

### ISSUE THREAD

**Issue by AdanC1b3r** [oxen-io/session-ios#890]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-ios/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The risks of not changing session nodes frequently in Session Messenger are relatively low, as the session network is designed to be decentralized and secure. However, there are some potential risks to be aware of:

Reduced reliability: if nodes are not changed frequently, it could lead to reduced reliability of the session network. This could result in messages being delayed or not delivered at all.

Increased vulnerability: If the same nodes are used for an extended period of time, it could make the network more vulnerable to attacks. However, the Session network is designed to be decentralized, with no single point of failure, which reduces the risk of attacks.

Limited scalability: If the same nodes are used for an extended period of time, it could limit the scalability of the Session network. However, the session network is designed to be highly scalable, with more than 1,500 community-operated servers currently routing session messages for more than 150,000 users.

In summary, while there are some potential risks associated with not changing session nodes frequently, the session network is designed to be decentralized and secure, which reduces the overall risk.

### Expected Behavior

I expect the nodes to be modified frequently, for example, every time the application is closed or opened.

### iOS Version

IOS 16.6

### Session Version

2.3.1
